aries-site/src/data/wcag/components.json

```diff
@@ -861,6 +861,518 @@
         "label": "Status Messages",
         "status": "not-applicable"
       }
+    ],
+    "textinput": [
+      {
+        "rule": "1.1.1",
+        "label": "Non-text content",
+        "status": "conditional",
+        "notes": "If an icon is provided, an aria-label can be added to the icon."
+      },
+      {
+        "rule": "1.2.1",
+        "label": "Audio-only and Video-only (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.2",
+        "label": "Captions (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.3",
+        "label": "Audio Description or Media Alternative (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.4",
+        "label": "Captions (Live)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.5",
+        "label": "Audio Description (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.6",
+        "label": "Sign Language (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.7",
+        "label": "Extended Audio Description (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.8",
+        "label": "Media Alternative (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.9",
+        "label": "Audio-only (Live)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.1",
+        "label": "Info and Relationships",
+        "status": "conditional",
+        "notes": "An aria attribute can be added."
```

Comment:
Could you elaborate more on the aria attribute you are envisioning being passed here?

In the textinput with suggestions example visually we get a clear indicator that there is a list of options available and that list is grouped in a drop. I think the screenreader announcement of this section could be improved so that a screenreader user better understands that the suggestions is a group of options

Reply:
I see, maybe I misunderstood the rule and was thinking an aria attribute could be added to the input in the cases where it's not in a FormField which could give more context to its purpose in absence of a "label".

I wasn't thinking about the suggestions experience that you mention. When I do VoiceOver I do get some indication that there's a menubox popup. I agree it could be improved though. Would you consider this "passed" because there is something" or "failed" because it's not the optimized?

Reply:
Kinda on the fence on this one but leaning towards 'failed' because I don't think we can confidently say that a screenreader user would have the same understanding of the component and a sighted user